File: Bijele.java
import java.util.Scanner;

public class Bijele 
{
    public static void main(String[] args) 
    {   
        Scanner sc = new Scanner(System.in);
        
        int king = sc.nextInt();
        int queen = sc.nextInt();
        int rooks = sc.nextInt();
        int bishops = sc.nextInt();
        int knights = sc.nextInt();
        int pawns = sc.nextInt();
        
        System.out.print(1-king+" ");
        System.out.print(1-queen+ " ");
        System.out.print(2-rooks+" ");
        System.out.print(2-bishops+" ");
        System.out.print(2-knights+" ");
        System.out.println(8-pawns);
    }
}

File: Bijele.py
ar1 = input().split()

king = 1 - int(ar1[0])
queen = 1-int(ar1[1])
rooks = 2-int(ar1[2])
bishops = 2-int(ar1[3])
knights = 2-int(ar1[4])
pawns = 8-int(ar1[5])

print(str(king) + " " + str(queen) + " " + str(rooks) + " " + str(bishops) + " " + str(knights) + " " + str(pawns))